### DISCUSSION TITLE
error with mpirun solid_dmft

### DISCUSSION THREAD

**Discussion by KeyuAN95** [TRIQS/solid_dmft#83] [General]
hello

I have installed the solid_dmft.
![Snipaste_2024-08-26_18-10-40](https://github.com/user-attachments/assets/a46dc097-537b-495f-bd9c-2debb7a940d4)

I ran $ mpirun solid_dmft    under the directory of /SVO_os_qe. I did not get "observables_imp0.dat".
There was a error following 

![Snipaste_2024-08-26_17-57-54](https://github.com/user-attachments/assets/c8a27319-73eb-4c9b-906f-5ddef8ad72e0)

How can I deal with it?

**Comment by the-hampel** [comment]
Hi @KeyuAN95,

this is indeed a simple configuration mistake inside of the pip package! I will try to make a new release in the next 24 hours. Thanks for reporting this. 

I will let you know once this is fixed.

Best,
Alex

**Reply by KeyuAN95** [reply]
Hi Alex,

How about the new release?
I am looking forward to hearing from you.

Best,
Keyu

**Reply by the-hampel** [reply]
Hey @KeyuAN95,
I just made a new release with a new pypi package: https://pypi.org/project/solid-dmft/3.3.1/ . I just checked and the missing default.toml is now included. Let me know if this one now works. Thanks! 

Best,
Alex

**Reply by KeyuAN95** [reply]
Hey Alex,

Thank you so much. The new release can work.

Best,
Keyu
